src/components/organisms/Room/Participant.tsx

```diff
@@ -138,28 +138,28 @@ export const Participant: React.FC<ParticipantProps> = ({
     [audioTracks]
   );
 
+  const videoElementClasses = classNames("", {
```

Comment:
Why empty string? Can't hurt to have a proper name. That being said, it will become `parent_name--modifier`, not `parent--modifier`